### ISSUE TITLE
Cart missing on mobile

### ISSUE THREAD

**Issue by Bassgarden** [woocommerce/woocommerce#22436]
I'm using Woocommerce 3.5.3 for my website. On a computer any page on the website the Cart always shows up, however on the Mobile, tested in Safari and on Chrome the only time the Cart shows up is when adding a product to the cart, after clicking on a different page the link to the cart doesn't show up on the top or anywhere on the site. 
It was working fine on previous version. I'm wondering if it's a glitch on the Woocommerce or if there's any options to enable on my side?

<img width="867" alt="screen shot 2019-01-13 at 9 23 11 am" src="https://user-images.githubusercontent.com/46649593/51086432-f9d85580-1714-11e9-82c7-cff791177b5d.png">

## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate
- [x] The issue still exists against the latest `master` branch of WooCommerce on Github (this is **not** the same version as on WordPress.org!)
- [x] I have attempted to find the simplest possible steps to reproduce the issue
- [ ] I have included a failing test as a pull request (Optional)

## Steps to reproduce the issue

1.
2.
3.

## Expected/actual behavior

When I follow those steps, I see...

I was expecting to see...

## Isolating the problem

- [x] This bug happens with only WooCommerce plugin active
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/)
- [ ] I can reproduce this bug consistently using the steps above

## WordPress Environment

<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin here.
```
</details>

**Comment by Bassgarden** [comment]
`
### WordPress Environment ###

Home URL: http://treesofjoy.com
Site URL: http://treesofjoy.com
WC Version: 3.5.3
Log Directory Writable: ✔
WP Version: ❌ 4.9.9 - There is a newer version of WordPress available (5.0.3)
WP Multisite: –
WP Memory Limit: 264 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.0.33 - We recommend using PHP version 7.2 or above for greater performance and security. How to update your PHP version
PHP Post Max Size: 20 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.62.0
OpenSSL/1.0.2q

SUHOSIN Installed: –
MySQL Version: 5.7.24
Max Upload Size: 2 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 3.5.3
WC Database Prefix: wp_
Total Database Size: 25.80MB
Database Data Size: 19.39MB
Database Index Size: 6.41MB
wp_woocommerce_sessions: Data: 1.55MB + Index: 0.06MB
wp_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_order_items: Data: 0.15MB + Index: 0.08MB
wp_woocommerce_order_itemmeta: Data: 0.95MB + Index: 0.74MB
wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB
wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.01MB
wp_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB
wp_woocommerce_log: Data: 0.00MB + Index: 0.00MB
wp_commentmeta: Data: 0.00MB + Index: 0.01MB
wp_comments: Data: 0.66MB + Index: 0.32MB
wp_elfsight_instagram_feed_widgets: Data: 0.00MB + Index: 0.00MB
wp_links: Data: 0.00MB + Index: 0.00MB
wp_options: Data: 3.89MB + Index: 0.37MB
wp_postmeta: Data: 7.44MB + Index: 2.49MB
wp_posts: Data: 0.60MB + Index: 0.14MB
wp_redirection_404: Data: 1.09MB + Index: 0.38MB
wp_redirection_groups: Data: 0.00MB + Index: 0.00MB
wp_redirection_items: Data: 0.00MB + Index: 0.01MB
wp_redirection_logs: Data: 1.09MB + Index: 0.49MB
wp_revslider_css: Data: 0.09MB + Index: 0.00MB
wp_revslider_layer_animations: Data: 0.00MB + Index: 0.00MB
wp_revslider_navigations: Data: 0.00MB + Index: 0.00MB
wp_revslider_sliders: Data: 0.02MB + Index: 0.00MB
wp_revslider_slides: Data: 0.07MB + Index: 0.00MB
wp_revslider_static_slides: Data: 0.00MB + Index: 0.00MB
wp_rg_form: Data: 0.00MB + Index: 0.00MB
wp_rg_form_meta: Data: 0.01MB + Index: 0.00MB
wp_rg_form_view: Data: 0.05MB + Index: 0.03MB
wp_rg_incomplete_submissions: Data: 0.00MB + Index: 0.00MB
wp_rg_lead: Data: 0.18MB + Index: 0.02MB
wp_rg_lead_detail: Data: 0.42MB + Index: 0.42MB
wp_rg_lead_detail_long: Data: 0.00MB + Index: 0.00MB
wp_rg_lead_meta: Data: 0.00MB + Index: 0.00MB
wp_rg_lead_notes: Data: 0.00MB + Index: 0.00MB
wp_termmeta: Data: 0.00MB + Index: 0.01MB
wp_terms: Data: 0.00MB + Index: 0.01MB
wp_term_relationships: Data: 0.01MB + Index: 0.02MB
wp_term_taxonomy: Data: 0.00MB + Index: 0.01MB
wp_usermeta: Data: 1.07MB + Index: 0.72MB
wp_users: Data: 0.05MB + Index: 0.05MB
wp_wc_download_log: Data: 0.00MB + Index: 0.00MB
wp_wc_webhooks: Data: 0.00MB + Index: 0.00MB
wp_yoast_seo_links: Data: 0.00MB + Index: 0.00MB
wp_yoast_seo_meta: Data: 0.00MB + Index: 0.01MB

### Post Type Counts ###

attachment: 214
nav_menu_item: 9
oembed_cache: 1
page: 12
post: 16
product: 126
revision: 131
shop_order: 1027
shop_order_refund: 12

### Security ###

Secure connection (HTTPS): ❌
					Your store is not using HTTPS. Learn more about HTTPS and SSL Certificates.
Hide errors from visitors: ✔

### Active Plugins (16) ###

Gravity Forms: by rocketgenius – 2.2.1.2
Anti-spam: by webvitaly – 5.3
Auction Nudge: by Joe Hawes – 6.2
Black Studio TinyMCE Widget: by Black Studio – 2.6.5
Elfsight Instagram Feed CC: by Elfsight – 3.1.1
Imsanity: by Exactly WWW – 2.4.1
WPBakery Page Builder: by Michael M - WPBakery.com – 5.4.5
Redirection: by John Godley – 3.7.1
Slider Revolution: by ThemePunch – 5.4.6.3.1
UPS WooCommerce Shipping: by Xadapter – 3.9.0 – Not tested with the active version of WooCommerce
USPS WooCommerce Shipping: by Xadapter – 3.1.13 – Not tested with the active version of WooCommerce
WooCommerce Print Invoice & Delivery Note: by Tyche Softwares – 4.4.7 – Not tested with the active version of WooCommerce
WooCommerce: by Automattic – 3.5.3
Yoast SEO: by Team Yoast – 9.4
WP Fastest Cache Premium: by Emre Vona – 1.4.3
WP Fastest Cache: by Emre Vona – 0.8.8.9

### Settings ###

API Enabled: ✔
Force SSL: ✔
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-catalog (exclude-from-catalog-2)
exclude-from-search (exclude-from-search)
featured (featured)
featured (featured-2)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-3 (rated-3-2)
rated-4 (rated-4)
rated-5 (rated-5)


### WC Pages ###

Shop base: #5 - /shop/
Cart: #6 - /cart/
Checkout: #7 - /checkout/
My account: ❌ Page does not contain the shortcode.
Terms and conditions: ❌ Page not set

### Theme ###

Name: Bridge Child
Version: 1.0.0
Author URL: http://www.qodethemes.com/
Child Theme: ✔
Parent Theme Name: Bridge
Parent Theme Version: 13.0
Parent Theme Author URL: http://www.qodethemes.com/
WooCommerce Support: ✔

### Templates ###

Archive Template: Your theme has a woocommerce.php file
you will not be able to override the woocommerce/archive-product.php custom template since woocommerce.php has priority over archive-product.php. This is intended to prevent display issues.

Overrides: bridge/woocommerce/cart/cart-empty.php version 3.1.0 is out of date. The core version is 3.5.0
bridge/woocommerce/cart/cart-shipping.php version 3.2.0 is out of date. The core version is 3.5.0
bridge/woocommerce/cart/cart-totals.php
bridge/woocommerce/cart/cart.php version 3.1.0 is out of date. The core version is 3.5.0
bridge/woocommerce/cart/shipping-calculator.php version 3.2.0 is out of date. The core version is 3.5.0
bridge/woocommerce/checkout/form-coupon.php version 2.2 is out of date. The core version is 3.4.4
bridge/woocommerce/checkout/thankyou.php
bridge/woocommerce/content-product.php version 3.0.0 is out of date. The core version is 3.4.0
bridge/woocommerce/content-single-product.php version 3.0.0 is out of date. The core version is 3.4.0
bridge/woocommerce/global/form-login.php version 2.1.0 is out of date. The core version is 3.3.0
bridge/woocommerce/global/quantity-input.php version 3.2.0 is out of date. The core version is 3.4.0
bridge/woocommerce/loop/add-to-cart.php version 3.0.0 is out of date. The core version is 3.3.0
bridge/woocommerce/loop/loop-start.php version 2.0.0 is out of date. The core version is 3.3.0
bridge/woocommerce/loop/pagination.php version 2.2.2 is out of date. The core version is 3.3.1
bridge/woocommerce/loop/rating.php
bridge/woocommerce/loop/result-count.php version 3.0.0 is out of date. The core version is 3.3.0
bridge/woocommerce/loop/sale-flash.php
bridge/woocommerce/myaccount/form-login.php version 3.2.0 is out of date. The core version is 3.5.0
bridge/woocommerce/myaccount/form-lost-password.php version 3.0.0 is out of date. The core version is 3.5.2
bridge/woocommerce/myaccount/my-address.php
bridge/woocommerce/myaccount/my-orders.php
bridge/woocommerce/single-product/add-to-cart/external.php version 2.1.0 is out of date. The core version is 3.4.0
bridge/woocommerce/single-product/add-to-cart/grouped.php version 3.0.7 is out of date. The core version is 3.4.0
bridge/woocommerce/single-product/add-to-cart/simple.php version 3.0.0 is out of date. The core version is 3.4.0
bridge/woocommerce/single-product/add-to-cart/variable.php version 3.0.0 is out of date. The core version is 3.4.1
bridge/woocommerce/single-product/related.php
bridge/woocommerce/single-product/sale-flash.php
bridge/woocommerce/single-product/tabs/additional-information.php
bridge/woocommerce/single-product/tabs/description.php
bridge/woocommerce/single-product/tabs/tabs.php
bridge/woocommerce/single-product/up-sells.php

Outdated Templates: ❌
					
					
						Learn how to update
`

**Comment by kloon** [comment]
This is due to how your theme is coded. I suggest you contact your theme developers and have them look into this for you. If you switch to a default theme like Storefront you will see that the cart is displayed even when no items are added to it.

![](http://cld.wthms.co/S0Colc+)
